guardian/homebrew-devtools
# feat: Add Oracle to split tunnel VPN connection

## What does this change?


Adds Oracle to the list of domains for `gvpn`. It's worth noting that `dc2-finprod-01.dc2.gnm.int` is where _I_ got directed to for authentication. I'm not sure if this is true for all attempts...

## How to test


- Checkout this branch
- Run `./scripts/run-vpn.sh`
- Open `https://oracle.dmz.gnl:9089/` and observe being able to connect to Oracle

## How can we measure success?


We're able to connect to Oracle when using `gvpn` to connect to the VPN.

## Images


![image](https://user-images.githubusercontent.com/836140/130751384-a722a0dc-82dc-4b5a-a769-54aa6ab74bd7.png)
